app/models/event_log.rb

```diff
@@ -88,7 +88,7 @@ def send_to_splunk(*)
       src_ip: (self.ip_address_string if self.ip_address.present?),
       action: self.event,
       result: self.trailing_message,
-      http_user_agent: self.user_agent&.user_agent_string
+      http_user_agent: self.user_agent&.user_agent_string || self.user_agent_string,
```

Comment:
Is this the same thing again or did I miss something? `user_agent_string` appears to be delegated to `user_agent`:  https://github.com/alphagov/signon/blob/5b66fcbf425e35035bbafeee4fafdc3243b235d7/app/models/event_log.rb#L65

Reply:
Whoops, that would have been a bug.  Good catch